src/MusicStore.Standalone/project.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft"
   ],
   "description": "Music store application on ASP.NET 5",
-  "version": "1.0.3",
```

Comment:
It's strange that we have these samples versioned but this is a relic of the past I guess.